### ISSUE TITLE
404 Not Found when generated app.js requests wasm file when using Emscripten and static index.html

### ISSUE THREAD

**Issue by martijn-heil** [koute/stdweb#307]
When doing a `cargo web start --target=wasm32-unknown-emscripten` whilst also having a  `static` directory with a `index.html` and then opening the app in the browser, the generated `app.js` script tries to load the wasm at `/js/appnamehere.wasm`, but the wasm file isn't located there and thus it responds with a 404, resulting in abortion of the loading process.
The wasm file *can* however be requested at `/appnamehere.wasm`.

#### Known conditions so-far for bug to occur
- The build target must be `wasm32-unknown-emscripten`
- You must have a `static` directory with a `index.html` in it , either in the crate root or in the `src` folder

#### Steps to reproduce
```
cargo init
mkdir static
* create the static/index.html *
cargo web start --target=wasm32-unknown-emscripten
```
Then in your browser of choice open the developer tools, specifically the network section, and navigate to the app url.

#### in-browser console log
```
wasm streaming compile failed: TypeError: Response has unsupported MIME type app.js:1722:11
falling back to ArrayBuffer instantiation app.js:1723:11
both async and sync fetching of the wasm failed app.js:6260:5
both async and sync fetching of the wasm failed app.js:6261:5
failed to asynchronously prepare wasm: abort("both async and sync fetching of the wasm failed") at jsStackTrace@http://localhost:8000/js/app.js:1048:13
stackTrace@http://localhost:8000/js/app.js:1065:12
abort@http://localhost:8000/js/app.js:6271:44
getBinary@http://localhost:8000/js/app.js:1625:7
getBinaryPromise/<@http://localhost:8000/js/app.js:1639:16
app.js:1709:9
abort("both async and sync fetching of the wasm failed") at jsStackTrace@http://localhost:8000/js/app.js:1048:13
stackTrace@http://localhost:8000/js/app.js:1065:12
abort@http://localhost:8000/js/app.js:6271:44
getBinary@http://localhost:8000/js/app.js:1625:7
getBinaryPromise/<@http://localhost:8000/js/app.js:1639:16
app.js:6260:5
abort("both async and sync fetching of the wasm failed") at jsStackTrace@http://localhost:8000/js/app.js:1048:13
stackTrace@http://localhost:8000/js/app.js:1065:12
abort@http://localhost:8000/js/app.js:6271:44
getBinary@http://localhost:8000/js/app.js:1625:7
getBinaryPromise/<@http://localhost:8000/js/app.js:1639:16
app.js:6261:5 
```

#### index.html contents
```
<!doctype html>
<html>
  <head>
    <meta charset="utf-8">
  </head>
  <body>
    <noscript><p>You need Javascript support to use this web app.</p></noscript>
    <!-- The application -->
    <script src="js/app.js"></script>
  </body>
</html>
```

#### System information
- Running the latest nightly Rust
- rustup 1.15.0
- rustc 1.32.0-nightly (4a45578bc 2018-12-07)
- cargo 1.32.0-nightly (5e85ba14a 2018-12-02)
- cargo-web 0.6.21
- OS is Arch Linux

**Comment by martijn-heil** [comment]
Turns out you shouldn't load `/js/app.js` but `/yourcratenamehere.js`, as it says in the console, no idea how I missed that. Interestingly enough `/js/app.js` does have javascript contents, but it isn't loaded either when you load via `/youcratenamehere.js`.
I think I got the `<script src="js/app.js"></script>` from some example from stdweb, if I can find such an  error somewhere there I'll make a pull request to fix that there.
